### PR TITLE
large partial pep cleanup in cluster_seed

### PR DESCRIPTION
**Pull request by fchapoton**
### 📚 Description

This tickets fixes many pep8 style warnings in the modified file.

### 📝 Checklist

- [x] I have made sure that the title is self-explanatory and the description concisely explains the PR.
- [ ] I have linked an issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/combinat/cluster_algebra_quiver/cluster_seed.py [hunk 1]
```diff
@@ -960,7 +960,8 @@ def _repr_(self):
         """
         name = self._description
         if self._mutation_type:
-            if type( self._mutation_type ) in [QuiverMutationType_Irreducible,QuiverMutationType_Reducible]:
+            if type(self._mutation_type) in [QuiverMutationType_Irreducible,
+                                             QuiverMutationType_Reducible]:
```
Comment (mkoeppe) [inline src/sage/combinat/cluster_algebra_quiver/cluster_seed.py]: rewrite using `isinstance`?

File: src/sage/combinat/cluster_algebra_quiver/cluster_seed.py [hunk 2]
```diff
@@ -3387,7 +3384,7 @@ def mutation_class_iter(self, depth=infinity, show_depth=False,
         Finite type examples not considered up to equivalence::
 
             sage: it = S.mutation_class_iter(up_to_equivalence=False)
-            sage: len( [ T for T in it ] )
+            sage: len( [T for T in it] )
             84
```
Comment (mkoeppe) [inline src/sage/combinat/cluster_algebra_quiver/cluster_seed.py:3388]: still too many spaces

File: src/sage/combinat/cluster_algebra_quiver/cluster_seed.py [hunk 3]
```diff
@@ -3594,7 +3591,7 @@ def cluster_class_iter(self, depth=infinity, show_depth=False, up_to_equivalence
         Finite type examples not considered up to equivalence::
 
             sage: it = S.cluster_class_iter(up_to_equivalence=False)
-            sage: len( [ T for T in it ] )
+            sage: len( [T for T in it] )
             84
```
Comment (mkoeppe) [inline src/sage/combinat/cluster_algebra_quiver/cluster_seed.py:3595]: still too many spaces

File: src/sage/combinat/cluster_algebra_quiver/cluster_seed.py [hunk 4]
```diff
@@ -3794,7 +3791,7 @@ def b_matrix_class(self, depth=infinity, up_to_equivalence=True):
         if depth is infinity and not self.is_mutation_finite():
             raise ValueError('The B-matrix class can - for infinite mutation types - only be computed up to a given depth')
 
-        return [ M for M in self.b_matrix_class_iter( depth=depth, up_to_equivalence=up_to_equivalence ) ]
+        return list(self.b_matrix_class_iter( depth=depth, up_to_equivalence=up_to_equivalence ))
```
Comment (mkoeppe) [inline src/sage/combinat/cluster_algebra_quiver/cluster_seed.py:3795]: too many spaces

**Comment by fchapoton** [comment]
thanks, done

**Review by mkoeppe — APPROVED** [review]

**Pull request merged**